grunttasks/copy.js

```diff
@@ -51,6 +51,20 @@ module.exports = function (grunt) {
           src: ['*.js']
         },
         {
+          cwd: '<%= yeoman.app %>/scripts/vendor/jwcrypto/',
```

Comment:
It might be nice to have a `requireOnDemand` task that loads the require config, looks for the paths of the modules that are required on demand, and updates the `copy` task as needed that way there is less manual work to do whenever a new script is loaded dynamically.